tests/framework/properties.py

```diff
@@ -102,6 +102,15 @@ def host_linux_version_tpl(self):
         """Host Linux version major.minor, as a tuple for easy comparison"""
         return tuple(int(x) for x in self.host_linux_version.split("."))
 
+    @property
+    def host_linux_version_metrics(self):
+        """Host Linux version to be reported in metrics"""
+        return (
+            "upstream"
```

Comment:
I know I was the one to suggest "upstream" when we talked about it in the office the other day, but how about "next" instead? That's what the upstream tree is called ("linux-next")

Reply:
Yeah next sounds good! I'll update it to that now